sdk/cognitivelanguage/ai-language-text/src/transforms.ts

```diff
@@ -161,6 +166,13 @@ function toSentimentAnalysisResult(
   });
 }
 
+function toDynamicClassificationResult(
```

Comment:
nit: toDynamicClassificationResult**s**

Reply:
will leave it as is to stay consistent with the other similar helpers which are named the same way.